doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -170,9 +170,15 @@ compatibility constraints.
 Adding new relations and morphisms
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd::  Add Parametric Relation {* @binder } : (A t1 ... tn) (Aeq t′1 ... t′m) {? reflexivity proved by @term} {? symmetry proved by @term} {? transitivity proved by @term} as @ident
+.. cmd::  Add Parametric Relation {* @binder } : @one_term @one_term {? reflexivity proved by @one_term } {? symmetry proved by @one_term } {? transitivity proved by @one_term } as @ident
 
-   This command declares a parametric relation :g:`Aeq: forall (y1 : β1 ... ym : βm)`,
+   .. todo PR needs more work.  The first @one_term maps to :g:`relation (A t1 ... tn)`.
+      I guess the second maps to :g:`Aeq: forall (y1 : β1 ... ym : βm)`, but that
+      doesn't match the (Aeq t′1 ... t′m) in the cmd syntax before my changes.
+      And then not sure how :g:`(A : αi -> ... αn -> Type)` fits in at all, e.g.
+      what is α?
+
+   Declares a parametric relation :g:`Aeq: forall (y1 : β1 ... ym : βm)`,
    :g:`relation (A t1 ... tn)` over :g:`(A : αi -> ... αn -> Type)`.
 
```

Comment:
I think that's correct, although it's a bit strange to separate the `Aeq : forall ...` from the `relation` part.

Reply:
Well, except that \beta1 ... \betan should be \alpha1 ... \alphan here.